src/iwant_bot/start.py

```diff
@@ -82,14 +87,27 @@ def check_post_request(body):
         #return web.json_response(body=format_response(user_name + ' wants me to ' + text))
     elif trigger_word == 'whoami':
         return web.json_response(body=format_response('You are ' + user_name + ' with id ' + user_id))
+
+    elif trigger_word == 'coffee':
+        res = web.json_response(body=format_response(user_name + ' wants coffee on ' + timestamp))
+        print("Response will be send in 2 minutes")
+        future = asyncio.Future()
+        asyncio.ensure_future(waiter(future, user_name))
+        future.add_done_callback(resulter)
+        return res
+
     else:
         return web.json_response(body=format_response(user_name + ' wrote ' + text))
 
 
-app = web.Application()
+loopX = asyncio.get_event_loop()
+loopX.set_debug(0)
+
+app = web.Application(loop=loopX)
```

Comment:
i'm assuming that web.app does have its own loop, but better to know what I'm doing on which loop, redundant for now, but can I work with loop in web.app without specification?